### DISCUSSION TITLE
flux version 2.0.0-rc.1 ✗ logger is required when boostrapping with bitbucket-server

### DISCUSSION THREAD

**Discussion by simon00000** [fluxcd/flux2#3803] [Q&A] [answered]
Anyone know what '✗ logger is required' means?

$ flux bootstrap bitbucket-server  --registry=my.private-registry  --image-pull-secret=regcred  --owner=flux  --username=flux  --repository=my-reop  --hostname=bitbucket.my.domain.io  --token-auth  --path=my-repo  --ca-file=/usr/local/share/ca-certificates/my-ca.pem  --branch develop  --token-auth 
✗ logger is required

The same command works as desired with flux v0.41.2 but with flux v2.0.0-rc.1 I cannot get past the message

**Comment by makkes** [comment]
@simon00000 is the output you provided above the only line you get? If not, please paste the full output (redacting sensitive information) here.

**Reply by simon00000** [reply]
That is all I get, nothing else.

**Comment by nevotheless** [comment]
I'm just wanted to try out flux and getting the same message which isn't quite helping me a lot :D

**Comment by kingdonb** [comment, marked as answer]
I think this bug is fixed in Flux 2.0.0-rc.2 (which hasn't been released yet as of this writing)

* https://github.com/fluxcd/go-git-providers/pull/210
* https://github.com/fluxcd/flux2/pull/3821

You can track the Flux 2.0.0-rc.2 release progress here:

* https://github.com/fluxcd/flux2/issues/3842
